tests/e2e/multicluster/multicluster_primaryremote_test.go

```diff
@@ -0,0 +1,353 @@
+    global:
+      remotePilotAddress: %s`
+
+						remotePilotAddress, err := common.GetSVCLoadBalancerAddress(ctx, clPrimary, controlPlaneNamespace, "istio-eastwestgateway")
+						Expect(remotePilotAddress).NotTo(BeEmpty(), "Remote Pilot Address is empty")
+						Expect(err).NotTo(HaveOccurred(), "Error getting Remote Pilot Address")
+						remoteIstioYAML := fmt.Sprintf(RemoteYAML, version.Name, remotePilotAddress)
+						Log("Remote Istio CR: ", remoteIstioYAML)
+						By("Creating Remote Istio CR on Remote Cluster")
+						Expect(kubectl.CreateFromString(remoteIstioYAML, kubeconfig2)).To(Succeed(), "Remote Istio Resource creation failed on Remote Cluster")
+
+						// Set the controlplane cluster and network for Remote namespace
+						By("Patching the istio-system namespace on Remote Cluster")
+						Expect(
+							kubectl.Patch("",
+								"namespace",
+								controlPlaneNamespace,
+								"merge",
+								`{"metadata":{"annotations":{"topology.istio.io/controlPlaneClusters":"cluster1"}}}`,
+								kubeconfig2)).
+							To(Succeed(), "Error patching istio-system namespace")
+						Expect(
+							kubectl.Patch("",
+								"namespace",
+								controlPlaneNamespace,
+								"merge",
+								`{"metadata":{"labels":{"topology.istio.io/network":"network2"}}}`,
+								kubeconfig2)).
+							To(Succeed(), "Error patching istio-system namespace")
+
+						// To be able to access the remote cluster from the primary cluster, we need to create a secret in the primary cluster
+						// RemoteIstio resource will not be Ready until the secret is created
+						// Get the internal IP of the control plane node in Remote cluster
+						internalIPRemote, err := kubectl.GetInternalIP("node-role.kubernetes.io/control-plane", kubeconfig2)
+						Expect(internalIPRemote).NotTo(BeEmpty(), "Internal IP is empty for Remote Cluster")
+						Expect(err).NotTo(HaveOccurred())
+
+						// Wait for the RemoteIstio CR to be created, this can be moved to a condition verification, but the resource it not will be Ready at this point
+						time.Sleep(5 * time.Second)
+
+						// Install a remote secret in Primary cluster that provides access to the Remote cluster API server.
+						By("Creating Remote Secret on Primary Cluster")
+						secret, err := istioctl.CreateRemoteSecret(kubeconfig2, "remote", internalIPRemote)
+						Expect(err).NotTo(HaveOccurred())
+						Expect(kubectl.ApplyString("", secret, kubeconfig)).To(Succeed(), "Remote secret creation failed on Primary Cluster")
+					})
+
+					It("secret is created", func(ctx SpecContext) {
+						secret, err := common.GetObject(ctx, clPrimary, kube.Key("istio-remote-secret-remote", controlPlaneNamespace), &corev1.Secret{})
+						Expect(err).NotTo(HaveOccurred())
+						Expect(secret).NotTo(BeNil(), "Secret is not created on Primary Cluster")
+						Success("Remote secret is created in Primary cluster")
+					})
+
+					It("updates Remote Remote Istio CR status to Ready", func(ctx SpecContext) {
```

Comment:
Maybe this makes it more clear that you're referring to a resource name?
```suggestion
					It("updates Remote RemoteIstio CR status to Ready", func(ctx SpecContext) {
```

Reply:
typo: ˙Remote Remote˙